Documentation/install/upgrade.rst

```diff
@@ -320,6 +320,37 @@ Annotations:
 
 .. _1.7_required_changes:
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~
+
+In rare cases, Cilium 1.7.12 or earlier when configured with ENI IPAM mode
+could disrupt connectivity for the node when new ENI devices are created on the
+node. This version fixes the issue and introduces a new flag to handle
+compatibility. This flag is ``--egress-multi-home-ip-rule-compat``.
+
+The flag defaults to ``false`` meaning it will instruct Cilium to perform a
+migration to the new ENI datapath upon Cilium upgrade. This operation will
+modify the routing table to fix a collision that in rare cases can cause
+connectivity loss to the node.
```

Comment:
I'm not sure the second sentence adds anything now that the first paragraph explains the bug and what the flag is?

```suggestion
The flag defaults to ``false`` meaning it will instruct Cilium to perform a
migration to the new ENI datapath upon Cilium upgrade. 
```

Reply:
Good catch, I think this was a leftover from previous feedback and the msg kinda got duplicated.